webrtc.html

```diff
@@ -1755,6 +1762,36 @@ <h4>Set the RTCSessionDescription</h4>
                               </li>
                             </ol>
                           </li>
+                          <li>
+                            <p>If <var>description</var> is of type <code>"answer"</code>
+                            or <code>"pranswer"</code>, then run the following steps:</p>
+                            <ol>
+                              <li>
+                                <p>If the offer contained a request to send simulcast and
+                                the answer indicates that simulcast is not supported or
+                                desired by the answerer, then set the
+                                <code><a data-link-for="RTCRtpEncodingParameters">active</a></code>
+                                flag to <code>false</code> on all the <var>sendEncodings</var>
+                                except the first, in the <code><a>RTCRtpSender</a></code>
```

Comment:
"on all dictionaries in *transceiver*.[[Sender]].[[sendEncodings]] except the first one,"